test/Microsoft.AspNetCore.Mvc.Core.Test/Formatters/FormatterMappingsTest.cs

```diff
@@ -83,14 +84,16 @@ public void FormatterMappings_Wildcardformat(string format)
         {
             // Arrange
             var options = new FormatterMappings();
-            var expected = string.Format(@"The media type ""{0}"" is not valid. MediaTypes containing wildcards (*) " +
-                "are not allowed in formatter mappings.", format);
+            var expected = $@"The media type ""{format}"" is not valid. MediaTypes containing wildcards (*) " +
+                "are not allowed in formatter mappings.";
 
             // Act and assert
-            var exception = Assert.Throws<ArgumentException>(() => options.SetMediaTypeMappingForFormat(
-                "star",
-                MediaTypeHeaderValue.Parse(format)));
-            Assert.Equal(expected, exception.Message);
+            var exception = ExceptionAssert.ThrowsArgument(
+                () => options.SetMediaTypeMappingForFormat(
+                    "star",
+                    MediaTypeHeaderValue.Parse(format)),
+                null,
```

Comment:
Part of the same bug I mentioned [above](https://github.com/aspnet/Mvc/pull/5011/files#r70498753).